test/source/platform/util.ts

```diff
@@ -15,3 +16,8 @@ export const base64encode = (binary: string): string => {
 export const base64decode = (b64tr: string): string => {
   return Buffer.from(b64tr, 'base64').toString('binary');
 };
+
+export const iso2022jpToUtf = (content: Buf) => {
+  const decoder = new TextDecoder();
```

Comment:
This code runs under Node, and wouldn't work anyway. Better to put in `throw new Error('iso2022jpToUtf not implemented on node')`

Reply:
Thanks